SpaceCommander.podspec

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
+Pod::Spec.new do |s|
+  s.name         = "SpaceCommander"
+  s.version      = "1.0.0"
+  s.summary      = "[ SpaceCommander] provides tools which enable you to commit Objective-C code to a git repository using a unified style format."
+  s.homepage     = "https://github.com/square/spacecommander"
+  s.license      = { :type => 'Apache', :file => 'LICENSE' }
+  s.author       = { "iOS Team" => "ios@squareup.com" }
```

Comment:
I don't think external people can email this address. I'd make it `s.authors  = 'Square'`